compliance-web/src/hooks/useStaff.tsx

```diff
@@ -0,0 +1,86 @@
+import { useQuery } from "@tanstack/react-query";
+import { request } from "@/utils/axiosUtils";
+import { Position } from "@/models/Position";
+import { Permission } from "@/models/Permission";
+
+export interface MockUser {
+  id: number;
+  name: string;
+}
+
+const mockUsersList: MockUser[] = [
```

Comment:
If this is the mock for the dropdown list. You should call the Epic.Auth endpoint directly.
`
    {
        "id": "5bca57e4-1b39-498c-8872-732d3da4beaf",
        "first_name": "Shaelyn",
        "last_name": "Tolk",
        "email_address": "shaelyn.tolk@gov.bc.ca",
        "username": "0790439a8c0448ed94ebe98d759bce6a@idir"
    },`
You have to show the {first_name} {last_name} format and get the username as the selected value

Reply:
Yes, This for the name and other users dropdown list. I will connect to epic.auth for this.